src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Analyzers/ConvertToLibraryImportFixer.cs

```diff
@@ -656,7 +656,7 @@ SyntaxNode WrapInvocationWithHRExceptionThrow(SyntaxNode node, SyntaxGenerator g
                         }
                         else
                         {
-                            if (dllImportData.EntryPointName.LastOrDefault() != entryPointSuffix.Value)
+                            if (dllImportData.EntryPointName?.LastOrDefault() != entryPointSuffix.Value)
```

Comment:
Do we always populate this value? If so, can we remove the `?` on the data structure? If we are reusing the Roslyn data structure, I assume we are, do we want the `null` check here? Given that `hasEntryPointAttributeArgument` is true, does that mean we have a name and then use `!` instead?